cron.js

```diff
@@ -2,9 +2,9 @@ const mainscript = require("./index.js")
 const CronJob = require('cron').CronJob
 
 console.log('Before job instantiation')
-const job = new CronJob('0/2 * * * * *', function () {
+const job = new CronJob('0/5 * * * * *', function () {
```

Comment:
What does '0/5' mean?

Reply:
It stands for five seconds. I used it for testing.